pages/common/r2e.md

```diff
@@ -0,0 +1,24 @@
+# r2e
+
+> Forwards RSS feeds to your email address. More information: https://github.com/rss2email/rss2email.
+> Requires a configured sendmail or smtp setup.
```

Comment:
Perhaps this would read better?
```suggestion
> Forwards RSS feeds to an email address.
> Requires a configured `sendmail` or smtp setup.
> More information: <https://github.com/rss2email/rss2email>.
```

Reply:
For some weird reason my tldr client renders multiple lines as one, so I thought that was the normal format.